IsFailedUserDataProcessingActivity/handler.ts

```diff
@@ -75,22 +78,23 @@ export const IsFailedUserDataProcessing = (
             reason: "ERROR|tableService.retrieveEntity|Cannot retrieve entity"
           })
       )
-    )
-    .chain(maybeTableEntry =>
-      fromEither(
-        fromOption(
+    ),
+    TE.chainW(
+      flow(
+        E.fromOption(() =>
```

Comment:
ditto

Reply:
fixed in https://github.com/pagopa/io-functions-admin/pull/165/commits/afa68583dbaee263dd115e88ae0eaba20e9075de